tasks/postgres_upgrade.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+echo 'Checking pgsql version'
+CMD=$(sudo gitlab-psql --version)
+echo "version is ${CMD##* }"
+#9.2.18
+if [ "${CMD##* }" = "9.2.18" ]; then
```

Comment:
would it better to do a `-lt 10` here instead of hardcoding a 9.x?

Reply:
Not in this case. Refer to the link in the OP's PR.